R/entities_objects.R

```diff
@@ -0,0 +1,114 @@
+# Contains functions to parse the objects described here: 
+# https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/entities
+
+# <https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/entities#hashtags>
+hashtags <- function(x) {
+  if (NROW(x) == 0) {
+    return(data.frame(text = NA, indices = I(list(NA)), 
+                      stringsAsFactors = FALSE))
+  }
```

Comment:
I think this function would be clearer if written more symmetrically:

```R
if (nrow(x) == 0) {
   data.frame(...)
} else {
  i <- indices_vec(x$indices)
  data.frame(...)
}
```

Reply:
Done!